app/src/main/java/com/example/tung/moviedb_07/data/source/remote/api/service/MovieApi.java

```diff
@@ -0,0 +1,43 @@
+package com.example.tung.moviedb_07.data.source.remote.api.service;
+
+import com.example.tung.moviedb_07.data.model.GenreList;
+import com.example.tung.moviedb_07.data.model.Movie;
+import com.example.tung.moviedb_07.data.model.MovieList;
+import io.reactivex.Observable;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Created by tung on 6/13/17.
+ */
+
+public interface MovieApi {
+
+    @GET("movie/popular")
+    Observable<MovieList> getPopularMovies(@Query("api_key") String apiKey, @Query("page") int
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "api_key" appears 7 times in this file; the first occurrence is on line 18
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals